packages/nextjs/src/config/withSentryConfig.ts

```diff
@@ -103,6 +103,15 @@ function getFinalConfigObject(
     );
   }
 
+  if (process.env.TURBOPACK && !process.env.SENTRY_SUPPRESS_TURBOPACK_WARNING) {
```

Comment:
Should we restrict to only show this warning in dev mode? No strong feelings, was only wondering because the warning says things work fine in prod.

Reply:
Oh good point, if the user does only use --turbo for dev, everything will be fine, but if they use it for prod, it gets funky. Need to update the message.